streisand

```diff
@@ -116,18 +177,13 @@ EOF
   # Create the inventory file
   echo "$TEMPL" > inventories/inventory-existing
   # Invoke the Streisand playbook on the existing server inventory
-  echo; echo; ansible-playbook -i inventories/inventory-existing --extra-vars=@global_vars/vars.yml playbooks/existing-server.yml
-}
-
-# runs the validation role to validate global_vars/vars.yml
-function validate() {
-  echo; echo; ansible-playbook playbooks/validate.yml
+  run_genesis existing-server.yml inventories/intenvory-existing
```

Comment:
intenvory -> inventory

Reply:
After fixing the inventory filename, I'm also seeing the following error when choosing Option 8 and running against an existing server:

```
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'streisand_genesis_role' is undefined"}
```

Reply:
That second error was unrelated to this commit and is also happening on master. I just submitted PR #955 as a proposed fix.